.ci.yaml

```diff
@@ -29,6 +29,21 @@ platform_properties:
       os: Ubuntu
       cores: "8"
       device_type: none
+  linux_android_emu:
+    properties:
+      contexts: >-
+        [
+          "android_virtual_device"
+        ]
+      dependencies: >-
+        [
+          {"dependency": "android_sdk", "version": "version:34v3"},
+          {"dependency": "android_virtual_device", "version": "34"},
+          {"dependency": "open_jdk", "version": "version:17"}
+        ]
+      os: Ubuntu
+      cores: "8"
```

Comment:
We need `kvm: "1"` for emulators.

Reply:
All the 8 core bots have kvm but it doesn't hurt to add it.